documentation/install_from_source.markdown

```diff
@@ -8,6 +8,7 @@ layout: default
 [configure_heap]: ./configure.html#configuring-the-java-heap-size
 [module]: ./install_via_module.html
 [packages]: ./install_from_packages.html
+[running_tests]: ./CONTRIBUTING.html#running-the-tests
```

Comment:
I'm not entire sure if this is right.....I'm guessing the tool that deploys these docs just takes the markdown files and translates them to .html files?